accepted/calc-constants.md

```diff
@@ -15,12 +15,10 @@ This proposal adds support for constant names in CSS calculations.
   * [Degenerate Number](#degenerate-number)
 * [Syntax](#syntax)
   * [`CalculationExpression`](#calculationexpression)
-* [Types](#types)
-  * [Calculations](#calculations)
 * [Semantics](#semantics)
   * [`CalcValue`](#calcvalue)
 * [Serialization](#serialization)
-  * [Calculations](#calculations-1)
+  * [Calculations](#calculations)
```

Comment:
Do you use a tool to generate the TOC? I'm just curious because it sounds useful

Reply:
Yeah, this one: https://github.com/sass/sass/blob/main/tool/update-toc.ts